TestHMACAuthSessionPass is flaky

I found tests that failing from time to time, and I'm not sure if it caused by recent changes or was there before.

```
--- FAIL: TestHMACAuthSessionPass (0.00s)
	middleware_check_HMAC_test.go:144: Initial request failed with non-200 code, should have gone through!: 
```
Link to failed build: https://travis-ci.org/TykTechnologies/tyk/builds/196897411

Happened again, not the same test though:

```
=== RUN   TestHMACAuthSessionPassWithHeaderFieldLowerCase
[Jul 10 22:37:47]  WARN Legacy path detected! Upgrade to extended.
[Jul 10 22:37:48] ERROR hmac: Clock skew outside of acceptable bounds
[Jul 10 22:37:48]  INFO hmac: Authorization field missing or malformed origin=192.0.2.1:1234 path=/
--- FAIL: TestHMACAuthSessionPassWithHeaderFieldLowerCase (0.04s)
	mw_hmac_test.go:356: Initial request failed with non-200 code, should have gone through!: 
		 400
```

--- a/mw_hmac_test.go
+++ b/mw_hmac_test.go
@@ -20,7 +20,7 @@ const hmacAuthDef = `{
 	"api_id": "1",
 	"org_id": "default",
 	"enable_signature_checking": true,
-	"hmac_allowed_clock_skew": 1000,
+	"hmac_allowed_clock_skew": 5000,
 	"auth": {
 		"auth_header_name": "authorization"
 	},

